### PR TITLE
Set VisTK language correctly PERU-157

### PR DESCRIPTION
**Pull request by makmanalp**
screen shot 2016-04-25 at 6 16 39 pm

### CODE REVIEW

**Comment by QuinnLee** [comment]
looks good

File: app/components/vistk-scatterplot.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default Ember.Component.extend({
   }),
   scatter: computed('data.@each', 'dataType','eciValue','i18n.locale', function() {
     let eci = this.get('eciValue');
-    let lang = this.get('i18n.locale') === 'en-col' ? 'en_EN': 'es_ES';
+    let vistkLanguage = this.get('i18n').display === 'es' ? 'es_ES': 'en_EN';
```
Comment (QuinnLee) [inline app/components/vistk-scatterplot.js]: does 
```js
this.get('i18n.display')
```
work?

**Comment by makmanalp** [comment]
@QuinnLee yep, that does indeed work. What's the difference between `this.get('i18n.display')` and `this.get('i18n').display`? Is it about whether the output of that will get updated when `i18n` changes vs when only `display` changes?

**Comment by makmanalp** [comment]
@QuinnLee I also tried to move this logic into locale-setter.js but I got weird issues where events were being handled in a weird order, where in locale-setter the `i18n.addObserver('locale', function() ...` was being fired AFTER the graph was getting rerendered somehow, making the language always the opposite of what it needs to be. 👎  Didn't have time to fully figure out why though.

**Comment by QuinnLee** [comment]
not much of a difference. `this.get('i18n.display')` is safer just incase `i8n` is undefined.

**Pull request merged**